src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -12,6 +12,20 @@
 
 {% block scripts %}
     <script src="{{ asset('bundles/packagistweb/js/view.js')}}"></script>
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/styles/monokai_sublime.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/highlight.min.js"></script>
+    <script>
+      $(document).ready(function() {
+        hljs.configure({
+          tabReplace: 4,
+          languages: ['apache', 'bash', 'ini', 'javascript', 'json', 'nginx', 'php', 'sql', 'css', 'html']
```

Comment:
Hmm, i thought that it speeds the highlighting up by reducing the number of languages the auto-detection has to work on. Because it invokes auto-detection on multiple DOM elements for only 10 languages instead of 135 (`console.log(hljs.listLanguages());`).